p2-maven-plugin/src/main/java/org/eclipse/tycho/p2maven/repository/P2RepositoryManager.java

```diff
@@ -101,6 +102,12 @@ public IArtifactRepository getCompositeArtifactRepository(Collection<Repository>
 		return new ListCompositeArtifactRepository(childs, agent);
 	}
 
+	public IArtifactRepository createLocalArtifactRepository(Path location, String name, Map<String, String> properties)
+			throws ProvisionException {
+		return getArtifactRepositoryManager().createRepository(location.toUri(), name,
```

Comment:
I won't use the repository manager here but `SimpleArtifactFactory` directly. this will also avoid the need to remove the repos later from the manager because it will never be added.

Reply:
Done.